### ISSUE TITLE
Several issues regarding the reproduction of this work

### ISSUE THREAD

**Issue by wnillinw** [luhr2003/UniGarmentManip#4]
Dear author, after reading your paper, I would like to reproduce your work. According to your instructions, after configuring the experimental environment and running the training code you provided, I found that the dataset was missing. After reviewing your paper, the dataset was sampled from cloth3d. As a 2D image researcher, after downloading the training set of this dataset, I found myself at a loss. I am not sure what to do next, so I would like to ask for your guidance on this issue.
